packages/SwingSet/test/vat-admin/terminate/test-terminate.js

```diff
@@ -69,7 +98,8 @@ test('dispatches to the dead do not harm kernel', async t => {
       'w: p1 = before',
       `w: I ate'nt dead`,
       'b: p1b = I so resolve',
-      'b: p2b fails vat terminated',
+      'b: p2b fails Error: vat terminated',
+      'done: undefined',
```

Comment:
Why do we get `done: undefined`? 

I presume this is from
```
 const doneP = E(dude.adminNode).done();
```
in bootstrap-terminate.js.   Why is adminNode unprepared to handle `done()`, and why does it manifest like this?

Reply:
It's from `bootstrap-speak-to-dead.js`, line 44, which is catching the exception thrown by awaiting the `done()` promise returned two lines above (line 42).  This is in turn triggered by the call to `terminate()` to lines above that (line 40), which says:
```
      E(weatherwax.adminNode).terminate();
```
Since this code predates the API refactoring, there's no parameter given to `terminate()` (which is entirely permissible, so I left it as it was, though actually passing a value there would arguably be clearer, so maybe I shoudl do that while I'm making the other test tweaks that Brian asked for), meaning that the termination reason in this case is `undefined`, which is what the catch caught.